### DISCUSSION TITLE
Fish Shell with gruvbox-material autocomplete wrong color

### DISCUSSION THREAD

**Discussion by esn89** [ghostty-org/ghostty#3509] [Q&A] [answered]
Thanks for releasing Ghostty.  I have one issue:

The native autocomplete that comes with fish, with the gruvbox-material them has the following autocomplete highlight in Kitty:

[kitty](https://imgur.com/a/ZG3NZPl)

On Ghostty it looks like this

[ghostty](https://imgur.com/a/PYlnG9m)

Here is my Ghostty config:

```
font-family = "PragmataProMonoLiga Nerd Font Mono"
font-size = 15
theme = gruvbox-material

macos-titlebar-style = "hidden"
shell-integration = "fish"
```

**Comment by mitchellh** [comment]
Possibly just an upstream https://iterm2colorschemes.com issue. Do the colors in the config match?

**Reply by esn89** [reply]
You are right, they don't match and I just re-defined it them myself.

**Reply by Lira-tron** [reply]
@esn89 can you  please share your config file? I also had this issue

**Reply by Lira-tron** [reply]
For me this happens in zsh also

**Reply by esn89** [reply]
```bat ~/.config/ghostty/colors
palette = 0=#665c54
palette = 1=#ea6962
palette = 2=#a9b665
palette = 3=#e78a4e
palette = 4=#7daea3
palette = 5=#d3869b
palette = 6=#89b482
palette = 7=#d4be98
palette = 8=#928374
palette = 9=#ea6962
palette = 10=#a9b665
palette = 11=#d8a657
palette = 12=#7daea3
palette = 13=#d3869b
palette = 14=#89b482
palette = 15=#d4be98
background = 282828
foreground = d4be98
cursor-color = d4be98
selection-background = d4be98
selection-foreground = 282828
```

**Reply by 00-kat** [reply]
Please PR the fix upstream if you haven't already.

**Reply by esn89** [reply, marked as answer]
https://github.com/mbadolato/iTerm2-Color-Schemes/pull/526

**Comment by sghng** [comment]
The color scheme in Ghostty is still not fixed on my side. Do I need to manually perform some cleanup to make the fixes scheme work?

<img width="235" alt="image" src="https://github.com/user-attachments/assets/5d59256a-5f3c-4b97-819e-b147ba850247" />

**Reply by sghng** [reply]
@esn89 were you able to get the updated color scheme in Ghostty?

**Reply by pluiedev** [reply]
Please do not comment on old discussions. Open a new one instead.
